src/lib/cardiac_mechanics/SolverException.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file SolverException.hpp
+ * @brief Header file defining the base class for runtime exceptions
+ */
+
+#ifndef SOLVER_EXCEPTION_HPP
+#define SOLVER_EXCEPTION_HPP
+
+#include <deal.II/base/exceptions.h>
+#include <string>
+
+/**
+ * @class SolverException
+ * @brief Utility class for defining the base class for runtime exceptions.
+ */
+class SolverException : public dealii::ExceptionBase {
```

Comment:
I believe that the same functionality of this class is achieved through `dealii::ExceptionMsg`.

Reply:
Yes, 100%. In the beginning, I only wanted to give a clear and fast understanding of the issue by using a custom class name. Now it might or might not be displayed during exceptions making this choice useful or not. Will check it.